storm-client/test/jvm/org/apache/storm/grouping/partialKeyGrouping/PartialKeyGroupingTest.java

```diff
@@ -53,7 +62,7 @@ public void testChooseTasksFields() {
         PartialKeyGrouping pkg = new PartialKeyGrouping(new Fields("test"));
         WorkerTopologyContext context = mock(WorkerTopologyContext.class);
         when(context.getComponentOutputFields(any(GlobalStreamId.class))).thenReturn(new Fields("test"));
-        pkg.prepare(context, mock(GlobalStreamId.class), Lists.newArrayList(0, 1, 2, 3, 4, 5));
+        pkg.prepare(context, mock(GlobalStreamId.class), Lists.newArrayList(9, 8, 7, 1, 2, 3));
```

Comment:
Why are we changing this?  The test should still pass just fine with the original values, and if not it is a bug.  So why change them?

Reply:
I changed these because at one point during my refactoring the test passed purely by coincidence with consecutive, increasing numbers. I wanted to ensure that was not the case, although I don't remember the specifics of which code caused that.

Reply:
Sounds reasonable.  Just FYI in reality they are likely to be consecutive increasing numbers because we assign the IDs by component in a consecutive increasing fashion.

Reply:
I can add another test that does use consecutive numbers. I simply wanted to ensure the code did not require consecutive numbers.